GoogleMapsComponents/Maps/Places/Autocomplete.cs

```diff
@@ -2,10 +2,30 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Microsoft.JSInterop;
 
 namespace GoogleMapsComponents.Maps.Places
 {
-    public class Autocomplete
+    public class Autocomplete : IDisposable
     {
+        private readonly JsObjectRef _jsObjectRef;
+
+        public async static Task<Autocomplete> CreateAsync(IJSRuntime jsRuntime, MarkerOptions opts = null)
+        {
+            var jsObjectRef = await JsObjectRef.CreateAsync(jsRuntime, "google.maps.Marker", opts);
+            var obj = new Autocomplete(jsObjectRef);
+
+            return obj;
+        }
+
+        private Autocomplete(JsObjectRef jsObjectRef)
+        {
+            _jsObjectRef = jsObjectRef;
+        }
+
+        public void Dispose()
+        {
+            _jsObjectRef.Dispose();
```

Comment:
Please use null check. You could use syntax sugar like _jsObjectRef?.Dispose();

Reply:
This is copied code other classes... I should of branched this piece, because this is part of the autocomplete address search.

Thinking about it, I will see if I can just create a branch and push the DirectionsResult piece alone, and then include this bit in another branch.